overlord/ifacestate/ifacemgr.go

```diff
@@ -36,7 +45,13 @@ func Manager() (*InterfaceManager, error) {
 }
 
 // Connect initiates a change connecting an interface.
-func (m *InterfaceManager) Connect(plugSnap, plugName, slotSnap, slotName string) error {
+//
+func Connect(change *state.Change, plugSnap, plugName, slotSnap, slotName string) error {
+	summary := fmt.Sprintf(i18n.G("Connecting %s:%s to %s:%s"),
```

Comment:
I think you are right. Perhaps we could disable i18n checks so that .pot doesn't annoy the hell out of us (better yet - automate it) but do use i18n all around.